src/app/shared/libraries/libraries.module.ts

```diff
@@ -0,0 +1,16 @@
+import { CommonModule } from '@angular/common';
+import { NgModule } from '@angular/core';
+import { AccordionComponent } from '@app/shared/libraries/accordion/accordion.component';
+import { AccordionItemComponent } from '@app/shared/libraries/accordion-item/accordion-item.component';
+
+export const PIPES_AND_COMPONENTS = [
```

Comment:
rename this variable to just  COMPONENTS. I think we should the autocomplete also into the libray module